src/app/commands/first-quest/FirstQuest.ts

```diff
@@ -101,11 +120,50 @@ module.exports = class FirstQuest extends SlashCommand {
 		const { guildMember } = await ServiceUtils.getGuildAndMember(ctx);
 		let command: Promise<any>;
 		switch (ctx.subcommands[0]) {
+		case 'start':
```

Comment:
@tikisailor, you can wrap the entire Switch statement in a `try/catch` block. That should help with readability, while grabbing any errors that filter through from each part of the switch.

Reply:
done